girder/newt/server/__init__.py

```diff
@@ -55,7 +55,8 @@ def create_assetstore_from_event(event):
 def load(info):
 
     AssetstoreType.NEWT = 'newt'
-    events.bind('assetstore.adapter.get', 'newt', getAssetstore)
+    events.bind('girder.utility.assetstore_utilities.setAssetstoreAdapter',
+        'newt', getAssetstore)
```

Comment:
setAssetstoreAdapter is actually a [function](https://github.com/girder/girder/blob/master/girder/utility/assetstore_utilities.py#L64). So we want to remove the bind and call the function instead the register the assetstore. So it would be something like:

```
setAssetstoreAdapter(AssetstoreType.NEWT, NewtAssetstoreAdapter)
```